.github/workflows/check-links.yml

```
@@ -35,9 +35,11 @@ jobs:
           --exclude "https://fonts.gstatic.com"
           --exclude "https://pkgs.org"
           --exclude "https://www.fdsn.org/xml/station"
+          --exclude "https://www.fdsn.org/webservices/"
```

Ignore all URLs starting with "https://www.fdsn.org/"
```suggestion
          --exclude "https://www.fdsn.org/"
```

---

But sometimes, the main URL may be the same, but the sub-url changes.

---

The error message is:
```
error trying to connect: error:1416F086:SSL routines:tls_process_server_certificate:certificate verify failed:../ssl/statem/statem_clnt.c:1915: (unable to get local issuer certificate))
```
So, something wrong with their SSL certification. I believe all their URLs are "invalid".